modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -8466,7 +8467,7 @@ type Program {
   proposalStatus: ProposalStatus!
 
   "Principal Investigator"
-  pi: User
+  pi: ProgramUser
```

Comment:
Do we need it here? Wouldn't it be enough to have the `pi` in  `users`?

Reply:
That's a good point and I wondered about doing that.  The PI is kind of "special" and it seemed like it might be handy to have a direct reference to it instead of having to pick it out of the `users` list.  That said, I thought I'd end up needing it in a predicate but it turns out I didn't.